pkg/cli/cli.go

```diff
@@ -351,12 +344,6 @@ func validatePlugins(plugins ...plugin.Base) error {
 					pluginName, projectVersion, err)
 			}
 		}
-		// Check for duplicate plugin names. Names outside of a version can
```

Comment:
We still need to check for duplicates. We should not allow 2 plugins with the same name and version.